packages/uniforms-bridge-json-schema/src/JSONSchemaBridge.ts

```diff
@@ -102,13 +102,14 @@ export default class JSONSchemaBridge extends Bridge {
       return null;
     }
 
-    const nameParts = joinName(null, name);
+    const nameParts = joinName(null, name).map(joinName.unescape);
+    const unescapedName = joinName(...nameParts);
```

Comment:
`joinName` can handle arrays:
```suggestion
    const unescapedName = joinName(nameParts);
```